active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
I think the use case that @mortonjt explained and what is exposed in this RFC are 2 different things.

Just to give some intuition on why there are 2 different things, the use case presented by @mortonjt would require to load all sequences present in the system, which is an overkill, given that such search is information about the sample, not the sequence.